Cargo.toml

```diff
@@ -25,6 +25,9 @@ serde_derive = "1.0.147"
 tempfile = "3.5.0"
 tinytemplate = "1.2.1"
 toml = "0.5.9"
+vhost-user-backend = "0.15.0"
+virtiofsd = { version = "1.11.1", default-features = false, git = "https://gitlab.com/d-e-s-o/virtiofsd.git", branch = "topic/different-caps-backend" }
```

Comment:
Does it still need to be pinned to your fork?

Reply:
That's for https://gitlab.com/virtio-fs/virtiofsd/-/merge_requests/260